src/Microsoft.SqlTools.ServiceLayer/DacFx/Contracts/DeploymentOptions.cs

```diff
@@ -436,9 +440,22 @@ public DeploymentOptions(DacDeployOptions options)
             // Setting Display names for all dacDeploy options
             SetDisplayNameForOption();
 
```

Comment:
Instead of copying the same two calls in both constructors, having a helper method that gets called in both places will allow for logic to be reused.

Reply:
Or even better, one should call the other (less specific calls more specific with appropriate defaults)

Reply:
Actually the first method is not more required and already removed in other main PR, this PR will merge later that PR, Thank you :)